build.rs

```diff
@@ -43,7 +46,7 @@ fn main() {
         println!("cargo:rustc-cfg=armv8m_main");
     }
 
-    if target.ends_with("-eabihf") {
-        println!("cargo:rustc-cfg=has_fpu");
+    if native_compilation {
+        println!("cargo:rustc-cfg=native");
```

Comment:
Is removing the `has_fpu` setting deliberate?

Reply:
whoops, this is definitely wrong. Thanks